tests/test_config_node.py

```diff
@@ -94,3 +96,20 @@ def test_add_cylc_install_to_rose_conf_node_opts(rose_conf, cli_conf, expect):
         'already in `rose-suite.conf`.'
     )]
     assert result.state == ''
+
+
+def test_dump_rose_log(monkeypatch, tmp_path):
+    # Pin down the results of the function used to provide a timestamp.
+    def fake(*arg, **kwargs):
+        return '18151210T0000Z'
+    monkeypatch.setattr(
+        DateTimeOperator, 'process_time_point_str', fake
+    )
+
+    node = ConfigNode()
+    node.set(['opts'], 'A B C')
+
+    dump_rose_log(tmp_path, node)
+
+    assert (tmp_path / 'log/18151210T0000Z-rose-suite.conf').read_text() == \
+        'opts=A B C\n'
```

Comment:
Curious as the dumper strips the `opts` in normal usage (expected), perhaps test with `[env]`.

Reply:
Do you want both, or `[env]` instead?

Reply:
I think just `[env]` because I don't think we would expect this example to work from the CLI?